### PR TITLE
test: document how to manually run Ubuntu CI stuff

### PR DESCRIPTION
**Pull request by mrc0mmand**
Every time I need it I have to first relearn autopkgtest and find where
all the necessary stuff lives, so let's document it somewhere close to
systemd for (at least) future me.

---

@bluca @mbiebl PTAL, as I'm not very experienced with things around autopkgtest, so maybe I'm speaking gibberish :-)

### CODE REVIEW

**Review by bluca — APPROVED** [review]
Looks great, thank you!

**Comment by mrc0mmand** [comment]
(Quick push fixing just a typo)

File: test/README.testsuite [hunk 1]
```diff
@@ -163,3 +163,48 @@ places:
...
+And finally run the autopkgtest itself:
+
+# autopkgtest -o logs *.deb systemd/ \
+              --timeout-factor=3 \
+              --test-name=boot-and-services \
+              --shell-fail \
+              -- autopkgtest-virt-qemu autopkgtest-focal-amd64.img
+
+where --test-name= is the name of the test you want to run/debug. The
+--shel-fail option will pause the execution in case the test fails and shows
```
Comment (bluca) [inline test/README.testsuite]: `shell`

Comment (mrc0mmand) [reply test/README.testsuite]: :facepalm:

**Comment by evverx** [comment]
@mrc0mmand thanks!

> the Ubuntu CI artifacts would benefit from some improvements

:-) It would also benefit from coverage reports, running the tests under ASan/UBsan, running a networkd testsuite that actually gets updated when networkd gets updated and other stuff that well-maintained CI services like CentOS CI do.

**Pull request merged**